web/controllers/registration_controller.ex

```diff
@@ -6,11 +6,11 @@ defmodule Pairmotron.RegistrationController do
   plug :scrub_params, "user" when action in [:create]
 
   def new(conn, _params) do
-    render conn, changeset: User.registration_changeset(%User{})
+    render conn, changeset: User.profile_changeset(%User{})
   end
 
   def create(conn, %{"user" => user_params}) do
-    changeset = User.registration_changeset(%User{}, user_params)
+    changeset = User.profile_changeset(%User{}, user_params)
```

Comment:
So I was creating a third user changeset called `profile_changeset`, and I realized that it looked exactly like `registration_changeset`. Rather than just reuse registration_changeset when there is no registration taking place, I thought it made more sense to just use profile_changeset as the name for both.